portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,66 @@
   width: 650px;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
 }
+
+p.artist{
+    color:#5e7059;
+    line-height:30px;
+}
+.container{
+    display:flex;
+    height:125px;
+    width:300;
+    flex-wrap:wrap;
+
+    
+    
+}
+.artist{
+    margin:20px;
+    height:20px;
+    width: 120px;
+    font-weight:bold;
+    color:#000000;
```

Comment:
Optional: throughout this file, alphabetize the attributes within a class.

This is a rule we have at Google to help make a css file more readable, but it varies from company to company, so you can ignore this comment  if want. Having the attributes in alphabetical order makes it easier to find something if you're looking for it, but on the flip side, some attributes only make sense when paired together, and you may loose that with alphabetical